package.json

```diff
@@ -8,9 +8,9 @@
     "test": "test"
   },
   "scripts": {
+    "build": "gulp build",
     "test": "gulp test",
-    "preinstall": "npm install -g typings",
```

Comment:
Migrated to local typings install and removed from `preinstall` to `contrib` scripts since this is only required during development.